test/views/SubmitFormViewSpec.scala

```diff
@@ -188,5 +189,74 @@ class SubmitFormViewSpec extends ViewBaseSpec {
         elementText("label[for=box1]") shouldBe "Swm Blwch 1 TAW a godwyd gennych ar werthiannau a chyflenwadau eraill"
       }
     }
+
+    "rendering the submit_form page with the bta links partial" should {
```

Comment:
You're already testing this content in BtaNavigationLinksTemplateSpec.scala so in this spec, you should only test that passing some HTML as a parameter then puts that HTML on the page